### DISCUSSION TITLE
Update fail (2.5.285)!

### DISCUSSION THREAD

**Discussion by flambio** [requarks/wiki#5850] [Error / Bug Report] [answered]
Update via web interface from version 2.5.285 to 2.5.291 fails! How can I resolve?

**Comment by ghost** [comment]
Yesterday I tried to update via upgrade companion as well, but it seems like the version is still the same. I tried upgrading from 2.5.281 to 2.5.291. When clicking the Upgrade button, it had me wait for a couple of minutes. Then when the bar was finished nothing really happened. No Error Message or anything. I checked the version in the administration area, but there it remained the same.

In case of a display error, is there a way to read the version of wiki.js from the SSH command line?

Also, I checked the container within SSH and it got created 16 hours ago, so I assume that the container only got restarted?

Any help with this? 

We are running Wiki.js on Docker and Ubuntu 20.04.

**Comment by ghost** [comment]
> Update via web interface from version 2.5.285 to 2.5.291 fails! How can I resolve?

Can you also maybe provide some Info on how it actually failed? Might be easier to break down then. Was it the same situation as mine?

**Comment by flambio** [comment]
https://github.com/requarks/wiki/discussions/5664

**Reply by ghost** [reply]
Thanks! Manually it worked now!

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/en/troubleshooting#unable-to-upgrade-from-the-administration-area
